src/test/java/com/wikia/webdriver/testcases/discussions/CreatingReplyTests.java

```diff
@@ -116,55 +77,29 @@ private void userOnDesktopMustBeLoggedInToUseReplyCreator(final ReplyCreator rep
     anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MercurySubpages.REGISTER_PAGE);
   }
 
+  private void userOnMobileMustBeLoggedInToUseReplyCreator(final ReplyCreator replyCreator) {
+    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MercurySubpages.JOIN_PAGE);
+  }
+
   private void anonymousUserOnReplyEditorClickIsRedirectedTo(final ReplyCreator replyCreator, final String urlFragment) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "anonymousUserOnReplyEditorClickIsRedirectedTo" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
it is used